### DISCUSSION TITLE
ResultList not displaying results

### DISCUSSION THREAD

**Discussion by redlinecodes** [appbaseio/reactivesearch#1998] [Q&A]
I'm pretty sure I have all the config in there and 'name' is a valid field in the index.  What does display is the stats and the pagination but not the actual results.

NOTE: I am not performing a search, I'm just trying to render the results page to do some minor styling changes from the default.

`import { ResultList, ReactiveList } from '@appbaseio/reactivesearch';`
`const {ResultListWrapper} = ReactiveList;`

```
<ResultListWrapper>
        <ResultList key={item._id}>
          <ResultList.Content>
            <ResultList.Title>
              <div>{item.name}</div>
            </ResultList.Title>
          </ResultList.Content>
        </ResultList>
      </ResultListWrapper>
```

**Comment by SavvyShah** [comment]
The given snippet doesn't contain any error by the description you've shared.
Are there any errors in the developer console?

To debug easily, please share a codesandbox.

**Reply by redlinecodes** [reply]
I actually fixed it thanks to the docs and about 30 minutes - sometimes I just need to slow down and not be in such a rush :)
